src/Personal/actionCreator.js

```diff
@@ -33,6 +36,49 @@ export function logout() {
   };
 }
 
+export function loginStart() {
+  return {
+    type: actionType.LOGIN_START,
+  };
+}
+
+export function loginSuccessful(payload) {
+  return {
+    type: actionType.LOGIN_SUCCESSFUL,
+    payload,
+  };
+}
+
+export function loginFailed(payload) {
+  return {
+    type: actionType.LOGIN_FAILED,
+    payload,
+  };
+}
+
+export const requestLogin = (userInfo) => (dispatch) => {
+  dispatch(loginStart());
+  fetch(`${BACKEND_URL}/sessions`, {
+    method: 'POST',
+    mode: 'cors',
+    headers: {
+      'Content-Type': 'Application/json',
+    },
+    body: JSON.stringify(userInfo),
+  }).then((response) => response.json()).then((response) => {
+    if (response.accessToken !== '' && response.accessToken !== undefined) {
+      dispatch(loginSuccessful({
+        email: userInfo.email,
+        accessToken: response.accessToken,
+      }));
+    } else {
+      dispatch(loginFailed({ message: response.message }));
+    }
+  }).catch((error) => {
+    dispatch(loginFailed({ message: error }));
```

Comment:
we shouldn't display native error messages, the user can do nothing with it. Let's say something "Somethign went wrong, please try again later"